src/guide/typescript-support.md

```diff
@@ -150,7 +189,7 @@ const Component = defineComponent({
     // needs an annotation
     greeting(): string {
       return this.message + '!'
-    }
+    },
```

Comment:
```suggestion
    }
```

Reply:
When I saw this document for the first time, I realized that there is something unusual about this code, which is different from other codes.

First of all, it is mentioned here, it writes a comma less, if there is a missing comma between the method and method of the JS object declaration, it will throw a syntax error. This is such a small defect, so I am handy Just fixed it.

Then when I re-read this code, I found more problems. It is different from other code styles in the document. In addition to the comma above, there are two omitable commas below (according to the style of other codes, it should Is omitted), but this does not cause a syntax error, so I didn't notice it at first.

```ts
import { defineComponent } from 'vue'

const Component = defineComponent({
  data() {
    return {
      message: 'Hello!'
    }
  },
  computed: {
    // needs an annotation
    greeting(): string {
      return this.message + '!'
    }     // <--------------------------------------------- There should be a comma here

    // in a computed with a setter, getter needs to be annotated
    greetingUppercased: {
      get(): string {
        return this.greeting.toUpperCase();
      },
      set(newValue: string) {
        this.message = newValue.toUpperCase();
      },  // <--------------------------------------------- There should not be a comma here
    },    // <--------------------------------------------- There should not be a comma here, too
  }
})
```

I think they should be corrected, either here or in the next PR.